src/main/resources/assets/create/lang/zh_cn.json

```
@@ -1751,6 +1820,11 @@
 	"create.ponder.cogwheel.text_1": "齿轮会将动力传递至临近的齿轮",
 	"create.ponder.cogwheel.text_2": "以此方式连接的齿轮，旋转方向相反",
 
+	"create.ponder.cogwheel_casing.header": "包裹齿轮",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
	"create.ponder.cogwheel_casing.header": "装壳齿轮",
```

我顺便改一下1.12的。